cog/core/downtime.py

```
@@ -0,0 +1,97 @@
+            "start": datetime.strftime(self.start, DATETIME_FORMAT),
+            "end": datetime.strftime(self.end, DATETIME_FORMAT),
+            "is_restored": self.is_restored,
+        }
+
+    def marked_as_restored(self) -> Downtime:
+        self.is_restored = True
+        return self
+
+
+def get_downtime_list() -> list[Downtime]:
+    with open(DOWNTIME_PATH, "r", encoding="utf-8") as file:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)